packages/infra/src/constructs/stack.ts

```diff
@@ -52,9 +78,9 @@ export class HookStack extends Stack {
     );
 
     // Deny access to resources that have a falsy `aws-codedeploy-hooks` tag.
-    beforeAllowTrafficHook.addToRolePolicy(
+    hookFunction.addToRolePolicy(
       new aws_iam.PolicyStatement({
-        actions: ['*'],
+        actions,
```

Comment:
Any downside to leaving the `*` for these denies?

Reply:
It was blocking creation of log groups for the lambda. Any other ideas?